CONTRIBUTING.md

```diff
@@ -29,7 +29,7 @@ Before creating bug reports, please check [this list](#before-submitting-a-bug-r
 * **Check the [FAQs on the official website](https://poetry.eustace.io)** for a list of common questions and problems.
 * **Check that your issue does not already exist in the [issue tracker](https://github.com/sdispater/poetry/issues)**.
 
-#### How do I submit a bug report
+#### How do I submit a bug report?
```

Comment:
Remain title case here too?

Reply:
I didn't change this, just added the missing question mark